src/sincere_singularities/modules/order_queue.py

```diff
@@ -81,11 +131,15 @@ async def discard_order(self, order_id: str) -> None:
         with suppress(KeyError):
             del self.orders[order_id]
 
+        # Wait 10-20 Seconds as an order Cooldown
+        order_timeout = random.randint(10, 20)
+        await asyncio.sleep(order_timeout)
+        await self.spawn_order()
+
     async def stop_orders(self) -> None:
         """Stop All Orders (when stopping the game)."""
         self.running = False
-        # TODO: Make sure these cant fail (or catch specific Errors)
-        with suppress(Exception):
+        with suppress(HTTPException, NotFound):
```

Comment:
Maybe use `suppress` for the following two lines individually, so if the first one raises the second one still runs.